packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx

```diff
@@ -182,6 +182,17 @@ export class CommandBarBase extends BaseComponent<ICommandBarProps, {}> implemen
     return <CommandButtonType {...commandButtonProps as IButtonProps} />;
   };
 
+  private _onButtonClick(item: ICommandBarItemProps): (ev: React.MouseEvent<HTMLButtonElement>) => void {
+    return ev => {
+      if (item.inactive) {
+        return;
+      }
```

Comment:
Is this also consistent with the old code? I think it'd be good to add a unit test for this too...

Reply:
Let me double check. I pulled that check in as it was used in the past.

Reply:
inactive is deprecated. it'll be removed next major release anyway.